Sources/CreateOptions/ConfigOptions.swift

```diff
@@ -568,6 +568,18 @@ public struct ConfigOptions: ParsableConfiguration {
         @Option public var filenameTemplate: String = "%0.swift"
     }
 
+    @Option public var enums: Enums
+
+    // sourcery: document,
+    /// Options specifically related to generating enums of strings.
```

Comment:
Just a thought, but #184 might mean that we generated enums from integer properties too so maybe worth rewording this:

```suggestion
    /// Options specifically related to enums generated from schemas.
```